### PR TITLE
Always use the Yarn installer script to install Yarn.

### PR DESCRIPTION
**Pull request by mdno**
Signed-off-by: David Calavera <david.calavera@gmail.com>

### CODE REVIEW

**Comment by rybit** [comment]
I have no problems with this (aside form could you rebase?) 

out of curiosity, what is the problem with installing via npm?

**Comment by calavera** [comment]
> are we sure that this version of yarn is fully backwards compatible?

yes

@mdno do you mind fixing the conflicts?

File: Dockerfile [hunk 1]
```diff
@@ -114,23 +114,25 @@ USER root
...
+
+RUN curl -o- -L https://yarnpkg.com/install.sh > /usr/local/bin/yarn-installer.sh
+
 # Install node.js
 USER buildbot
 RUN git clone https://github.com/creationix/nvm.git ~/.nvm
 
 ENV ELM_VERSION=0.17.1
-ENV YARN_VERSION=0.17.10
+ENV YARN_VERSION=0.18.1
```
Comment (KyleAMathews) [inline Dockerfile]: This is extraneous now right?

**Comment by brycekahle** [comment]
Looks like this landed with https://github.com/netlify/build-image/commit/f9932e0c5cb91a9e0309412c3664aaf0da70af92

**Pull request closed without merging**